File: Data Structures & Algorithms/reverse-string/submission-0.java
class Solution {
    public void reverseString(char[] s) {
        char[] r = new char[s.length];
        for(int i=0;i<s.length;i++){
            r[i] = s[s.length-i-1];
        }
        for(int i=0;i<s.length;i++){
            s[i]=r[i];
        }
        return;
    }
}

File: Data Structures & Algorithms/majority-element/submission-0.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        int n = nums.size();
        unordered_map<int, int> freq;
        for(int i=0;i<n;i++){
            freq[nums.at(i)]++;
            if(freq[nums.at(i)]>n/2){
                return nums.at(i);
            }
        }
        return 0;
    }
};